README.md

```diff
@@ -0,0 +1,14 @@
+# Todoly
+ `Todoly` is a task organizer
+
+## Usage
+1.Build <br/>
+
+```
+docker build -t <image name> .
+```
+2.Run
+```
+docker run --rm -i --name <container name> -t <image name>
```

Comment:
Same here, put the full command already ready. Don't expect people to replace container and image name here. 



```
docker run --rm -it --name todoly-app todoly:latest
```

(-i and -t can be combined like above)